[DOCS-1902] Add rate limit details for Dedicated Cloud and Self-Managed (#2197)
wandb/docs: snippets/en/_includes/rate-limits-defaults-and-notification.mdx

The following default limits help maintain platform stability:

| Limit | Default | Scope |
|-------|---------|-------|
| Filestream requests per second | 500 | Project |

> | Filestream size | 120 MB per second | Project |

Reviewer: ```suggestion
| Filestream ingestion per second | 120 MB | Project |
```

> | Filestream requests per run per second | 2 | Run |

Reviewer: ```suggestion
| Filestream requests per second | 2 | Run |
```
maybe redundant because the scope column

| Run creation requests per second | 80 | Project |

> When a limit is exceeded, the W&B SDK returns HTTP 429. You may see a message such as the following in your SDK logs:

Reviewer: Maybe we can add that filestream rate limits should never cause logging to crash or fail. When SDK gets a 429 response on a filestream request, it will backoff and retry the rate-limited request as-is, while subsequent updates are able to accumulate.

Reviewer: Run creation rate limits will block training.

Author: WDYT about the suggested edit below?

> When a limit is exceeded, the W&B SDK returns HTTP 429. You may see a message such as the following in your SDK logs:

Author: ```suggestion
When a limit is exceeded, the W&B SDK returns HTTP response `429`, and the message `HTTP 429: rate limited exceeded` appears in the SDK logs.

- Filesystem rate limits never cause logging to crash or fail. When the SDK receives a `429` response on a filestream request, it will back off and retry the rate-limited request as-is, while subsequent updates accumulate.

- Run creation rate limits block further training.
```
